.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -12,8 +12,10 @@ A clear and concise description of what the bug is.
 
 **To Reproduce**
 Steps to reproduce the behaviour:
-1. Provide image with which you observed the issue 
+
+1. Provide image with which you observed the issue (commonly named POC)
```

Comment:
Poc = proof of concept. This name is afaik only really used in the security research community though.
```suggestion
1. Provide the image with which you observed the issue (commonly named PoC)
```